examples/e1.20/DEVICE_INFO.json

```diff
@@ -85,15 +85,15 @@
       ]
     },
     {
-      "name": "current_personality",
+      "name": "personality",
```

Comment:
Do we want a field name and an optional field label maybe?

OLA currently uses one field for both (with some automagic conversion to the other).

I can certainly see the argument for flagging the piece of information the same if it's present in multiple PIDs. But if this was called personality and we also had personality count in this PID it's probably not as user friendly, especially given it's actually the personality number not a name or anything.

Reply:
Technically speaking they are not names or labels. They are unique keys that can be used for localisation and hopefully to distinguish matching fields across parameters. The value of this key is shown only to the user as a last resort, you would first check whether there is a "displayName" key in the schema and present that first, before using this and then arguably I would probably just use the PID...